.github/workflows/publish-website.yml

```diff
@@ -0,0 +1,28 @@
+name: publish-website
```

Comment:
When following the steps from PR #6, when running `mkdocs serve`, I am getting:

```
(venv) rsoaresd-mac:kubesaw.github.io rsoaresd$ mkdocs serve
WARNING -  Config value 'jobs': Unrecognised configuration name: jobs
WARNING -  Config value 'True': Unrecognised configuration name: True
WARNING -  Config value 'permissions': Unrecognised configuration name: permissions
WARNING -  Config value 'name': Unrecognised configuration name: name
ERROR   -  Config value 'site_name': Required configuration not provided.

Aborted with a configuration error!
```

Do we need this `site_name` or am I missing something?